tox.ini

```diff
@@ -40,7 +40,6 @@ commands =
     ./tox.cover.sh
 
 [testenv:lint]
-# recent versions of pylint do not support Python 2.6 (#97, #187)
```

Comment:
could we please not remove this comment? I'm gonna have really hard times tracking this back when py 2.6 support will be back...